source/includes/sharding-warning.rst

```diff
@@ -0,0 +1,13 @@
+
+.. warning::
+
+   Syncing large document to a sharded destination cluster can cause 
```

Comment:
[question]

Should this be plural: "documents"?

Reply:
Fixed